lib/mongodb.js

```diff
@@ -336,41 +342,41 @@ MongoDB.prototype.getDefaultIdType = function() {
 
 /**
  * Get collection name for a given model
- * @param {String} model Model name
+ * @param {String} modelName Model name
  * @returns {String} collection name
  */
-MongoDB.prototype.collectionName = function(model) {
-  var modelClass = this._models[model];
+MongoDB.prototype.collectionName = function(modelName) {
```

Comment:
Please extract renames into a standalone pull request. When renames are mixed with code changes that are changing behavior, it's easy to miss an important change during the review. Let's quickly land the renames first and them make this pull request much smaller in patch size.